apps/test-suite/tests/Fetch.ts

```diff
@@ -159,6 +159,32 @@ export function test({ describe, expect, it, ...t }) {
       }
       expect(error).not.toBeNull();
     });
+
+    it('should abort streaming request', async () => {
+      const controller = new AbortController();
+      setTimeout(() => controller.abort(), 500);
+      let error: Error | null = null;
+      try {
+        const resp = await fetch('https://httpbin.org/drip?numbytes=512&duration=2', {
```

Comment:
How do you know that this backend service will behave the same during each test? I expect some flaky-ness will happen. 
It could happen that the service is offline or is slow, so the abort function in the timeout will be called too soon.

Reply:
good call. i have the other pr #33558 to use a different server. should also update this pr. thanks!

Reply:
Okay, I hope that is sufficient. One other solution to consider is to mock the backend call and response, so the response will always work, no matter the internet connection. MSW is a standard nowadays.

Reply:
> One other solution to consider is to mock the backend call and response, so the response will always work, no matter the internet connection. MSW is a standard nowadays.

i was looking for a proper mock, which would still receive and reply network requests. i feel that setup mocks in either js or native side is like not real. a mock server is much better, and MSW looks decent. thanks for sharing this. if the new server is not stable still, i would go MSW for a try.

Reply:
Happy to have helped! 👍